services/account_type/resource_owner.yaml

```diff
@@ -0,0 +1,190 @@
+            Action: 
+              - 'sts:AssumeRole'
+            Condition:
+              StringEquals:
+                'sts:ExternalId': !Sub '"${ExternalId}"'
+
+      Path: "/cloudzero/"
+
+  RolePolicy:
+    Type: AWS::IAM::Policy
+    Condition: CreateResources
+    Properties: 
+      PolicyName: cloudzero-resource-owner-policy
+      PolicyDocument: 
+        Version: 2012-10-17
+        Statement: 
+        - Sid: ProtectAccountInfo1
+          Effect: Deny
+          Action: aws-portal:*Account
+          Resource: "*"
+        - Sid: AccessCZLogGroups1
+          Effect: Allow
+          Action: logs:*
+          Resource:
+          - arn:aws:logs:*:*:log-group:/cz*:*:*
+          - arn:aws:logs:*:*:log-group:/cz*
+        - Sid: CreateVPCFlowLogs1
+          Effect: Allow
+          Action:
+          - logs:CreateLogStream
+          - logs:CreateLogGroup
+          - logs:PutSubscriptionFilter
+          - logs:PutRetentionPolicy
+          - ec2:CreateFlowLogs
+          - ec2:DescribeFlowLogs
+          Resource: "*"
+        - Sid: PassReactorRole1
+          Effect: Allow
+          Action:
+          - iam:PassRole
+          Resource: !GetAtt Role.Arn
+        - Sid: ReducedViewOnly1
+          Action:
+          - apigateway:GET
+          - apigateway:OPTIONS
+          - apigateway:HEAD
+          - athena:List*
+          - batch:ListJobs
+          - budgets:View*
+          - cloudformation:List*
+          - cloudformation:Get*
+          - cloudformation:Describe*
+          - cloudsearch:List*
+          - cloudwatch:List*
+          - cloudwatch:GetMetricData
+          - codebuild:ListBuilds*
+          - codestar:List*
+          - codestar:Verify*
+          - cognito-idp:List*
+          - cognito-identity:ListIdentities
+          - cognito-sync:ListDatasets
+          - connect:List*
+          - config:List*
+          - datapipeline:GetAccountLimits
+          - discovery:List*
+          - dlm:Get*
+          - dms:List*
+          - dynamodb:List*
+          - dynamodb:Describe*
+          - ecr:List*
+          - ec2:Get*
+          - ec2:Describe*
+          - es:Describe*
+          - es:List*
+          - elasticbeanstalk:List*
+          - elasticfilesystem:Describe*
+          - elasticmapreduce:List*
+          - elastictranscoder:List*
+          - events:ListRuleNamesByTarget
+          - events:ListTargetsByRule
+          - gamelift:List*
+          - glacier:List*
+          - health:*
+          - importexport:List*
+          - inspector:List*
+          - iot:List*
+          - iam:Get*
+          - iam:List*
+          - iam:Simulate*
+          - lambda:List*
+          - lambda:GetAccountSettings
+          - lambda:GetAlias
+          - lambda:GetEventSourceMapping
+          - lambda:GetFunctionConfiguration
+          - lambda:GetPolicy
+          - machinelearning:Describe*
+          - opsworks:Describe*
+          - opsworks-cm:Describe*
```

Comment:
FWIW I have a ticket entered to get you guys some more information to update this list.  I got pre-empted to work on other things though so I'm not likely to get there anytime soon